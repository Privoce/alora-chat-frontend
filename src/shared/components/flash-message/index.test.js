import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import {
	FlashMessageComponent
} from 'shared/components';

describe('<FlashMessageComponent />', () => {
	const modelI = {
		message: 'teste message',
		error: true
	};

	const getWrapper = (model) => {
		return shallow((
			<FlashMessageComponent {...model} />
		));
	};

	it('renders correctly', () => {
		const wrapper = getWrapper(modelI);
		expect(toJson(wrapper)).toMatchSnapshot();
	});
});
