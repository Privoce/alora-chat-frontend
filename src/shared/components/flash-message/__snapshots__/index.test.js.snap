// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FlashMessageComponent /> renders correctly 1`] = `
<div
  className="flash-message-wrapper error"
  style={
    Object {
      "margin": undefined,
      "maxWidth": undefined,
    }
  }
>
  <span
    className="flash-message"
  >
    teste message
  </span>
  <div
    className="button-container"
  >
    <ButtonComponent
      height={26}
      link={true}
      onClick={[Function]}
      type="button"
      width={26}
    >
      <IconComponent
        fill="#ffffff"
        height={26}
        icon="close"
        width={26}
      />
    </ButtonComponent>
  </div>
</div>
`;
